src/Calculator.ts

```diff
@@ -213,7 +211,63 @@ export function calculateMeasureReports(
         });
       }
 
-      createMeasureScore(measure, group, detail, report);
+      // if the measure definition has stratification, add group.stratifier
+      const stratifier = measure.group?.find(g => g.id == detail.groupId)?.stratifier;
+      if (stratifier) {
+        group.stratifier = [];
+        stratifier.forEach(s => {
+          const reportStratifier = <R4.IMeasureReport_Stratifier>{};
+          reportStratifier.code = s.code ? [s.code] : [];
+          const strat = <R4.IMeasureReport_Stratum>{};
+          // use existing populations, but reduce count as appropriate
+          strat.population = JSON.parse(JSON.stringify(group.population));
```

Comment:
Q: what's the need for the parse/stringify combo here?

Reply:
Deep copy + matching type structures that aren't actually the same type